README.md

```diff
@@ -781,6 +782,40 @@ Future.do(function*(){
 //> "Oh no! Failed to load /message"
 ```
 
+### Sanctuary
+
+When using this module with [Sanctuary][21], you may want to create a more
+explicit BinaryType type for Future using [Sanctuary Def][31]. For convenient
+creation of a Sanctuary BinaryType, the functions `extractLeft` and
+`extractRight` are provided.
+
+```js
+const FutureType = $.BinaryType(
+  Future.name,
+  Future.isFuture,
+  Future.extractLeft,
+  Future.extractRight
+);
+
+//...
+
+const toErrorMessage = def(
+  'toErrorMessage',
+  {},
+  [FutureType($.Error, a), FutureType($.Error, a)],
+  Future.mapRej(e => e.message)
+);
```

Comment:
I'm confused by this function. What is it meant to demonstrate, exactly? Also, should the return type be `FutureType($.String, a)` rather than `FutureType($.Error, a)`?